pkg/storaged/pages.jsx

```diff
@@ -863,7 +863,7 @@ export const StoragePage = ({ location, plot_state }) => {
     const page = get_page_from_location(location);
 
     return (
-        <Page id="storage" className={"no-masthead-sidebar" + (client.in_anaconda_mode() ? " storage-anaconda" : "")}>
+        <Page id="storage" className={"no-masthead-sidebar" + (client.in_anaconda_mode() ? " anaconda" : "")}>
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22419-5962424f-20250909-103027-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)